server/views/partials/temporaryAbsenceCardMacro.njk

```diff
@@ -0,0 +1,28 @@
+{% macro temporaryAbsenceCard(temporaryAbsence, loopIndex) %}
+    <div class="app-card app-card--with-aside">
+        <div class="app-card__content" style="margin-left: 115px">
+            <header class="app-card__header">
+                <h2 class="app-card__title" data-qa="temporaryAbsence-title-{{loopIndex}}">{{ temporaryAbsence.lastName }}, {{ temporaryAbsence.firstName }}</h2>
+                <span class="govuk-body">{{ temporaryAbsence.prisonNumber if temporaryAbsence.prisonNumber else "No prison number found" }}</span>
```

Comment:
This should never happen as this data is retrieved from NOMIS. 

On the choose prisoner page,  where it could happen then you can use the default filter 
```
temporaryAbsence.prisonNumber | default('No prison number found')